collect_app/src/main/java/org/odk/collect/android/tasks/InstanceGoogleSheetsUploader.java

```diff
@@ -273,6 +283,27 @@ private void insertRow(TreeElement element, String parentKey, String key, File i
         }
     }
 
+    /**
+     * Adds titles ending with "-altitude" or "-accuracy" that may have been added to the
+     * Sheet. Existing spreadsheets can start collecting altitude / accuracy from
+     * Geo location fields.
+     *
+     * @param sheetHeaders - Headers from the spreadsheet
+     * @param columnTitles - Column titles list to be updated with altitude / accuracy titles
+     */
+    private void addAltitudeAndAccuracyTitles(List<Object> sheetHeaders, List<Object> columnTitles) {
+        for (Object sheetTitle : sheetHeaders) {
+            String sheetTitleStr = (String) sheetTitle;
+            if (sheetTitleStr.endsWith(altitudeTitlePostfix) || sheetTitleStr.endsWith(accuracyTitlePostfix)) {
+                if (! columnTitles.contains(sheetTitleStr)) {
```

Comment:
Hey, someone else who likes a space after the 'not' operator!

Reply:
:) .. I'll blame it on a typo